sentry-android-replay/src/main/java/io/sentry/android/replay/viewhierarchy/ViewHierarchyNode.kt

```diff
@@ -282,7 +282,7 @@ sealed class ViewHierarchyNode(
                 (parent?.elevation ?: 0f) + view.elevation,
                 distance = distance,
                 parent = parent,
-                shouldRedact = false,
+                shouldRedact = options.experimental.sessionReplay.redactClasses.contains(view.javaClass.canonicalName),
```

Comment:
`m`: I think for completeness we should also have this check for `TextViewHierarchyNode` and  `ImageViewHierarchyNode`, but you can skip this if you don't have time right now

Reply:
Thanks, yes, that make sense.

Reply:
yeah but I meant we should change that, i.e. if you disable `redactAllText` but add `TextView` to `redactClasses` it should still redact textviews. But as I said you don't have to do it now, so it's fine to ignore this comment